src/utils/trackEvent.ts

```diff
@@ -35,17 +34,16 @@ const trackEvent = (event: any) => {
     return;
   }
 
-  axios({
+  fetch(`${VGS_COLLECT_KEEPER}/vgs`, {
     method: 'POST',
-    url: `${VGS_COLLECT_KEEPER}/vgs`,
-    data: payload,
+    body: payload,
   })
-    .then(() => {
-      return true;
-    })
-    .catch(() => {
-      return;
-    });
+  .then(() => {
+    return true;
+  })
+  .catch(() => {
+    return;
```

Comment:
just for the sake of consistency with Line 42: 
```suggestion
    return true;
```